tutor/commands/k8s.py

```diff
@@ -296,13 +297,18 @@ def start(context: K8sContext, names: List[str]) -> None:
     names = names or ["all"]
     for name in names:
         if name == "all":
-            # Create volumes
-            kubectl_apply(
-                context.root,
-                "--wait",
-                "--selector",
-                "app.kubernetes.io/component=volume",
-            )
+            # Create volumes if exists
```

Comment:
nit: this comment can be updated to highlight when volumes are not created or vice versa.